lang/algol68g/Portfile

```diff
@@ -15,14 +14,55 @@ long_description \
 homepage            http://jmvdveer.home.xs4all.nl/algol.html
 master_sites        http://jmvdveer.home.xs4all.nl/
 
-checksums           rmd160  7dec7b8e81efae5261c8b1fd0abaa1ca18e5faf0 \
-                    sha256  bd499f90576a8d86008316f55e026e08abe2c9bda930e7a00cd0bec5b2b2dc44
+checksums           rmd160  7849acdadd91ca6a9cac1537bfaac102e1969de7 \
+                    sha256  d407b36c4b5e984a63b24125d4a13f34771bebb65d55058375d85ddb0ac31b5b \
+                    size    621615
 
-depends_lib         port:gsl port:readline port:ncurses
+compiler.c_standard 1999
 
-configure.args      --disable-postgresql
+depends_build       port:autoconf \
+                    port:automake \
+                    port:libtool
+
+depends_lib         port:gmp \
+                    port:gsl \
+                    port:mpfr \
+                    port:ncurses \
+                    port:r \
+                    port:readline
+
+patchfiles          patch-a68glib.diff \
+                    patch-alloc.diff \
+                    patch-includedir.diff
+
+post-patch {
+    reinplace "s|@@PREFIX@@|${prefix}|g" ${worksrcpath}/Makefile.am
+}
+
+use_autoreconf      yes
+
+configure.args      --disable-postgresql \
+                    --disable-plotutils
+
+platform darwin powerpc {
+    patchfiles-append \
+                    patch-darwin-ppc.diff
```

Comment:
Unless this patchfile will cause problems for non-PowerPC systems, please include it unconditionally. It is likely that the next person to update this port will not be using a PowerPC computer, and by including the patch unconditionally, it ensures that if the patch needs to be updated, that person will not inadvertently forget to do so.

Reply:
@ryandesign Thank you. This makes perfect sense.